test/js/geo/transform.test.js

```diff
@@ -141,7 +141,7 @@ test('vertex buffer', function(t) {
     t.test('is initialized', function(t) {
         var buf = new VertexBuffer();
         t.deepEqual(buf.index, 0);
-        t.deepEqual(buf.length, 32768);
+        t.deepEqual(buf.capacity, 8192);
```

Comment:
Length is zero here in the current implementation. In the old buffers they all defined a defaultLength, which we don't do anymore. I think this was for allocating memory, which is why I switched it to capacity.